README.md

```diff
@@ -5,24 +5,56 @@
 The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
 
 ### On macOS and Linux
+
 ```bash
 $ source setup.sh
 ```
+
 ### On Windows (Using PowerShell)
+
 ```powershell
 $ .\setup.ps1
 ```
+
 ### On Windows (Using Git Bash)
+
 ```bash
 $ source setup.sh --windows
 ```
 
 Once the setup script has completed and all packages have been installed, start the Flask app by running:
+
 ```bash
 $ flask run
 ```
 
+### Initial Setup for Checklist Appplication
+
+Add the following to the .env file.
+
+Add your Trello api details.
+
+1. TRELLOAPIKEY = 'yourAPIkey'
+2. TRELLOSERVERTOKEN = 'yourAPIkey'
+
+Add the following values to the ApiAccess.py folder.
+
+  TODOLISTURL = 'https://api.trello.com/1/lists/YOURTODOLISTID/cards/'
+  DOINGLISTURL = 'https://api.trello.com/1/lists/YOURDOINGLISTID/cards/'
+  DONELISTURL = 'https://api.trello.com/1/lists/YOURDONELISTID/cards/'
+
+  CARDSURL = 'https://api.trello.com/1/cards/'
+  LISTURL = 'https://api.trello.com/1/lists/'
+  TODOLISTID = 'YOURTODOLISTID'
```

Comment:
These List Ids would be better stored in the environment variables so that we're only adjusting the code in one place; it will also allow for easier overwriting of these values for tests.
The various URLs can then be hardcoded or pieced together based on these variables